courses/tasks.py

```diff
@@ -39,3 +47,71 @@ def subscribe_edx_course_emails(enrollment_id):
     if subscribed:
         enrollment.edx_emails_subscription = subscribed
         enrollment.save()
+
+
+@app.task
+def generate_course_certificates():
+    """
+    Task to generate certificates for courses.
+    """
+    now = now_in_utc()
```

Comment:
The body of this task is complex enough that it should be moved into an `api.py` method (or two).